README.md

```diff
@@ -90,6 +90,32 @@ If the core file requires other services or controls, we can mock them with an u
 Some modules use React; see below for more information.
 
 #### React
+
+##### Presentation only Components
+
+React Components included in the pattern library should be purely presentational. This is similar to thinking of them as being read-only. This will mean that in most cases React versions of patterns will have no state, they will be entirely driven by props given to them.
+
+The most direct observation of this principal is that without a wrapping Container Component, inputs like text boxes should appear to be non-interactive. Going so far that if typed into they will refuse to update (or more accurately, immediately be redrawn with their pre-typed content). Changes in input values should fire events rather than modifying the DOM directly as they would in traditional usage. For an example of this, see `ff_module-form-input`.
+
+##### JSX
+
+JSX syntax is available, and should be used whenever practical to replace calls to React.createElement() due to the transparency of the DOM structure it offers.
+
+##### Separate Template Files
+
+Separate template files are required if any of the following conditions are true for a given pattern:
+
+ - The pattern is JavaScript/React only. The absense of an XSL means a template is required to quickly see the shape of a patterns output.
+ - The pattern outputs more than one DOM element. As soon as the complexity of a pattern increases to contain more than one HTML DOM element a template is required to quickly see the shape of a patterns output.
```

Comment:
I think the need is captured better in your next point @AdamLynamFirefly 

> In most cases these templates should naturally grow from raw HTML files used to plan the pattern.

I think the templates that are sufficiently simple enough to build directly into React, then no problem. Those that are more complex then how about we set a rule that either:

- There must be a flat HTML or XSLT version built first
- There must be some documentation included in the main `.md` file that contains the raw HTML for the component, and sufficient explanation of states and any classes that get added. 

The only thing I'm trying to cover off here is the writing / reviewing of complex markup, and getting that right before including that in the React.